### ISSUE TITLE
[ELE-64] Show databases and schemas in dbt  on the elementary report

### ISSUE THREAD

**Issue by yu-iskw** [elementary-data/elementary#463]
**Is your feature request related to a problem? Please describe.**
We use dbt with BigQuery and have a lot of GCP projects and BigQuery datasets in each. As, the elementary report shows only table names corresponding to tests, it can get much better to show database (GCP projects) and schemas too.

![image](https://user-images.githubusercontent.com/1523515/206630725-60ddb45b-4171-4eb4-a8b9-826c77448b77.png)

**Describe the solution you'd like**

**Describe alternatives you've considered**
I don't think we have any alternative solutions.

**Additional context**
Nothing

**Would you be willing to contribute this feature?**
I would love to contribute.

<sub>[ELE-64](https://linear.app/elementary/issue/ELE-64/show-databases-and-schemas-in-dbt-on-the-elementary-report)</sub>

**Comment by oravi** [comment]
Hi @yu-iskw ! thanks so much for opening this issue! today do you have a single dbt project that creates tables / views in different GCP projects and datasets?

**Comment by yu-iskw** [comment]
Yes. We create tables and views in different GCP projects. And source tales as dbt sources exist in different GCP project due to data security. We separate GCP projects of sources from ones of models, since I don't want grant writable/editable permissions in GCP projects of sources to a Google service account to create models. But, the service account has permissions to read sources. So, it can run tests of both sources and models in different GCP projects. That's why I would like to show databases(GCP projects) and schemas(datasets).

**Comment by oravi** [comment]
Thanks @yu-iskw ! we added this task to our task management backlog so this issue will go through our product scoping and prioritization process now. Usually one of our considerations for prioritizing is demand from the community. We'll keep you updated once we have news about this issue
